packages/fuels-core/src/utils.rs

```diff
@@ -29,3 +30,14 @@ pub fn ident(name: &str) -> Ident {
 pub fn safe_ident(name: &str) -> Ident {
     syn::parse_str::<SynIdent>(name).unwrap_or_else(|_| ident(&format!("{}_", name)))
 }
+
+pub fn get_logs_hashmap(
```

Comment:
I'm not convinced by the name here

Reply:
changed to `log_type_lookup`

Reply:
Maybe something that is more descriptive of what the hashmap holds. 
`get_contract_id_log_id_mapping`?